internal/pkg/plugin/trello/trello.go

```diff
@@ -28,6 +28,25 @@ func CreateTrelloBoard(options *Options) (*TrelloItemId, error) {
 		return nil, err
 	}
 
+	exist, err := c.CheckBoardExists(options.Owner, options.Repo, options.KanbanBoardName)
+	if err != nil {
+		return nil, err
+	}
+
+	if exist {
+		log.Successf("Board already exists, owner: %s, repo: %s, kanbanName: %s.", options.Owner, options.Repo, options.KanbanBoardName)
```

Comment:
If the board has already existed, we do nothing here, so the `log.Successf()` isn't very suitable. I think `Info or Debug` log level is better.

Reply:
For the same reason, we can move the `log.Successf()` after the board/list is created successfully below.